pbft/pbft.hpp

```diff
@@ -101,6 +102,8 @@ namespace bzn
         void checkpoint_reached_locally(uint64_t sequence);
         void maybe_stabilize_checkpoint(const checkpoint_t& cp);
 
+        void handle_database_message(const bzn::message& json, std::shared_ptr<bzn::session_base> session);
```

Comment:
same.. reference session...

Reply:
same as above, this is a bzn::message_handler